CHANGELOG.md

```diff
@@ -101,4 +101,6 @@
 
 - Remove unused packages and update packages with vulnerabilities (INDIGO Sprint 210625, [!135](https://github.com/TeskaLabs/asab-webui/pull/135))
 
-- Fix microservices containers indentation in order to keep clean code style (INDIGO Sprint 210723 ,[!143](https://github.com/TeskaLabs/asab-webui/pull/143))
\ No newline at end of file
+- Fix microservices containers indentation in order to keep clean code style (INDIGO Sprint 210723 ,[!143](https://github.com/TeskaLabs/asab-webui/pull/143))
+
+- Update `getVersion` function in `webpack/common.js` to prevent app from crashing when there is no repository for the project (INDIGO Sprint 210723, [!!146](https://github.com/TeskaLabs/asab-webui/pull/146))
```

Comment:
@t-pyrope Small remark - you have 2 exclamation marks in the changelog PR number

```
[!!146]
```